docs/.vuepress/_snippets/install_os.md

```diff
@@ -11,9 +11,10 @@ Finally, you can follow the [Quickstart](#quickstart) to take it from here and c
 
 Run the following script to automatically detect the operating system and download Kuma:
 
-```sh
-curl -L https://kuma.io/installer.sh | bash -
-```
+<div class="language-sh">
+<pre><code>curl -L https://kuma.io/installer.sh | VERSION={{ $page.latestVersion }} bash -</code></pre>
```

Comment:
**Suggestion**: Since we’re using `sh` everywhere, let’s keep it consistent here as well.

```suggestion
<pre><code>curl -L https://kuma.io/installer.sh | VERSION={{ $page.latestVersion }} sh -</code></pre>
```

Reply:
Oh ok sure I'll do them right now.